src/Atomizer/Renderer.php

```diff
@@ -119,6 +119,90 @@ public function dropTable(Source $source, AbstractTable $table): void
         );
     }
 
+    /**
```

Comment:
Why is this in scope of this update?

Reply:
I just run the spiral fixer. Will exclude from the PR